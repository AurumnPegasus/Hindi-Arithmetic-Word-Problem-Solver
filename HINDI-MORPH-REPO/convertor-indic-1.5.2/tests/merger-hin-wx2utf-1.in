<Sentence id='1'>
1	((	NP	
1.1	((	NP	<fs etype="ne" esubtype="location"  >
1.1.1	badZOxe	NNP	<fs af='badZOxe,unk,,,,,,'  >
	))
1.2	se	PSP	<fs af='se,psp,,,,,,'  >
	))
2	((	NP	
2.1	((	NP	<fs etype="ne" esubtype="location"  >
2.1.1	baMbaI	NNP	<fs af='baMbaI,n,m,sg,3,d,0,0'  >
	))
	))
3	((	VGNF	
3.1	jAwe	VM	<fs af='jA,v,m,pl,any,,wA,wA'  >
	))
4	((	NP	
4.1	samaya	NN	<fs af='samaya,n,m,sg,3,d,0,0'  >
	))
5	((	NP	
5.1	((	NP	<fs etype="numx" value="1.0"  >
5.1.1	eka	QC	<fs af='eka,adj,any,any,,any,,' etype="numx" value="1.0"  >
	))
5.2	relave	NNC	<fs af='relave,n,m,sg,3,d,0,0'  >
5.3	steSana	NN	<fs af='steSana,n,m,sg,3,d,0,0'  >
	))
6	((	VGF	
6.1	hE	VM	<fs af='hE,v,any,sg,3,,hE,hE'  >
6.2	.	SYM	<fs af='.,punc,,,,,,'  >
	))
</Sentence>

