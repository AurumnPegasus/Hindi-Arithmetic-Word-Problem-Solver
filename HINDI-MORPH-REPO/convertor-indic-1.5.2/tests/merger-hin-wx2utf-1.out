<Sentence id='1'>			
1	((	NP	
1.1	((	NP	<fs  esubtype="location"  etype="ne">
1.1.1	बड़ौदे	NNP	<fs af='बड़ौदे,unk,,,,,,'>
	))
1.2	से	PSP	<fs af='से,psp,,,,,,'>
	))
2	((	NP	
2.1	((	NP	<fs  esubtype="location"  etype="ne">
2.1.1	बंबई	NNP	<fs af='बंबई,n,m,sg,3,d,0,0'>
	))
	))
3	((	VGNF	
3.1	जाते	VM	<fs af='जा,v,m,pl,any,,ता,wA'>
	))
4	((	NP	
4.1	समय	NN	<fs af='समय,n,m,sg,3,d,0,0'>
	))
5	((	NP	
5.1	((	NP	<fs  etype="numx"  value="1.0">
5.1.1	एक	QC	<fs af='एक,adj,any,any,,any,,' etype="numx"  value="1.0">
	))
5.2	रेलवे	NNC	<fs af='रेलवे,n,m,sg,3,d,0,0'>
5.3	स्टेशन	NN	<fs af='स्टेशन,n,m,sg,3,d,0,0'>
	))
6	((	VGF	
6.1	है	VM	<fs af='है,v,any,sg,3,,है,hE'>
6.2	.	SYM	<fs af='.,punc,,,,,,'>
	))
</Sentence>			
			
